finetuner-workflow/finetune-workflow.yaml

```diff
@@ -306,35 +362,37 @@ spec:
     inputs:
       parameters:
```

Comment:
`workflow.parameters.retokenize` exists and is passed as `retokenize` to the `model-tokenizer` template earlier in the workflow, but the template itself neither defines it as a parameter nor uses it at the moment. In some earlier commits there was a commented out `-retokenize` flag passed along with the args to the container, but it seems to have disappeared.

Reply:
Addressed somewhat in #186. The workflow's `retokenize` parameter is now correctly passed to the tokenizer, and defaults to `'true'` instead of `'false'`.

I agree that not retokenizing is spicy with the tokenizer's current false-negative rate when detecting changes. If more detailed metadata were saved alongside the tokenized dataset (e.g. in a sidecar `.json` file with the same name prefix) and used to check for differences, I think it would be safe to switch `retokenize` back to a default of `'false'`. Until then, I'd leave it as an optional optimization chosen by the workflow user.